README.md

```diff
@@ -251,13 +251,13 @@ method takes two arguments:
 A value that implements the Traversable specification must also
 implement the Functor and Foldable specifications.
 
-1. `t(u.sequence(f.of))` is equivalent to `u.map(t).sequence(g.of)`
+1. `t(u.traverse(x => x, F.of))` is equivalent to `u.traverse(t, G.of)`
 for any `t` such that `t(a).map(f)` is equivalent to `t(a.map(f))` (naturality)
```

Comment:
Or rather not `t :: Applicative F, Applicative G => F a -> G a` but `t :: F a -> G a`, in other words `t` works not with any two Applicatives but specifically with types `F` and `G`.

Maybe the law should say "... for any `t :: F a -> G a` such ...".